solr/solr-ref-guide/src/solrcloud-configuration-and-parameters.adoc

```diff
@@ -1,5 +1,5 @@
 = SolrCloud Configuration and Parameters
-:page-children: setting-up-an-external-zookeeper-ensemble, using-zookeeper-to-manage-configuration-files, zookeeper-access-control, collections-api, parameter-reference, command-line-utilities, solrcloud-with-legacy-configuration-files, configsets-api
+:page-children: setting-up-an-external-zookeeper-ensemble, using-zookeeper-to-manage-configuration-files, collections-api, parameter-reference, command-line-utilities, solrcloud-with-legacy-configuration-files, configsets-api
```

Comment:
A page can only be child of ONE parent. I feel zookeeper-ACL fits better under securing-solr than here

Reply:
Yes, I agree, it's a good move.